promptsource/templates/tau/scrolls/qmsum/templates.yaml

```diff
@@ -0,0 +1,59 @@
+dataset: tau/scrolls
+subset: qmsum
+templates:
+  2edf0197-e3d4-4870-bd95-178411e52e09: !Template
+    answer_choices: null
+    id: 2edf0197-e3d4-4870-bd95-178411e52e09
+    jinja: 'This is a summary of a meeting based on a query.
+    
+    Summary: {{ output }}
+    
+    The query given and the original meeting transcript\
+    are as follows: ||| {{ input }}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      languages:
+      - en
+      metrics:
+      - ROUGE
+      - BLEU
+      original_task: false
+    name: write_meeting
+    reference: ''
+  3bd082cb-4e28-4eb7-9fa2-dd03f1f86219: !Template
+    answer_choices: null
+    id: 3bd082cb-4e28-4eb7-9fa2-dd03f1f86219
+    jinja: 'Here is a summary of a meeting:
+
+    {{ output }}
+
+    The following is the question used to create the summary,\
+    followed by the meeting transcript itself: ||| {{ input }}'
```

Comment:
Not really different from the previous one no?

Reply:
Can remove these duplicate prompts!